src/react-app-env.d.ts

```diff
@@ -1 +1,19 @@
 /// <reference types="react-scripts" />
+
+export interface User {
+  id: number,
+  name: string,
+  username: string,
+  email: string,
+}
+
+export interface Todo {
+  id: number,
+  title: string,
+  completed: boolean,
+  userId: number,
+}
+
+export interface TodosWithUsers extends Todo {
+  user: User | null,
+}
```

Comment:
why you did not create `types` folder with files for each type?)

Reply:
As our mentors' said - it's ok to create types in that file